manifests/config/apache.pp

```diff
@@ -65,14 +65,18 @@
 # $https_vhost_options::      Direct options to apache::vhost for the https vhost
 #
 class foreman::config::apache(
+  Boolean $passenger = $::foreman::passenger,
   Stdlib::Absolutepath $app_root = $::foreman::app_root,
   Optional[String] $listen_on_interface = $::foreman::passenger_interface,
-  Optional[String] $ruby = $::foreman::passenger_ruby,
+  Optional[String] $passenger_ruby = $::foreman::passenger_ruby,
   String $priority = $::foreman::vhost_priority,
   Stdlib::Fqdn $servername = $::foreman::servername,
   Array[Stdlib::Fqdn] $serveraliases = $::foreman::serveraliases,
   Stdlib::Port $server_port = $::foreman::server_port,
   Stdlib::Port $server_ssl_port = $::foreman::server_ssl_port,
+  Stdlib::Httpurl $proxy_backend = "http://${::foreman::foreman_service_bind}:${::foreman::foreman_service_port}/",
+  Hash $proxy_params = {'retry' => '0'},
+  Array[String] $proxy_no_proxy_uris = ['/pulp', '/streamer', '/pub'],
```

Comment:
These are the ones Katello has. I'm wondering if this is needed or if there's a cleaner solution.

Reply:
`/pub` is probably required, `/pulp` is a wsgi mount, isn't it?

Reply:
Honestly, I don't know exactly. Some locations are mapped to files. It's a very complicated deployment. This was my "rather safe than sorry" solution.